hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/CallIdSequenceWithBackoff.java

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
I would call this a hack because you copy paste the code to fix it a bit different in another place. Having shared code saves a huge amount of time.